cfg/params.yaml

```
@@ -346,6 +346,49 @@
   usage: Param for exponential backoff algorithm, which is used to increase waiting time b/w two consecutive retries.
   default: 2
 
+- config-path: "gcs-retries.read-stall.enable"
+  flag-name: "enable-read-stall-retry"
+  type: "bool"
+  usage: >-
+    Enable/disable retry for stalled read-request based on dynamic timeout.
+  default: false
+  hide-flag: true
+
+- config-path: "gcs-retries.read-stall.initial-req-timeout"
+  flag-name: "initial-read-stall-req-timeout"
+  type: "duration"
+  usage: Initial value of the read-request dynamic timeout.
+  default: 20s
+  hide-flag: true
+
+- config-path: "gcs-retries.read-stall.max-req-timeout"
+  flag-name: "max-read-stall-req-timeout"
+  type: "duration"
+  usage: Upper bound of the read-request dynamic timeout.
+  default: 20m
+  hide-flag: true
+
+- config-path: "gcs-retries.read-stall.min-req-timeout"
+  flag-name: "min-read-stall-req-timeout"
```

following the naming convention that's been used throughout this file.
```suggestion
  flag-name: "read-stall-min-req-timeout"
```